src/index.mako.html

```diff
@@ -230,6 +230,8 @@
     </div>
 % endif
 
+    <div ga-controls3d ga-controls3d-ol3d="ol3d" ng-if="ol3d" ng-show="globals.is3dActive"></div>
```

Comment:
The nice thing about `ng-if` is that it initialize the directive only when the condition is true.
That means that the `ga-controls3d` is sure that the `scope.ol3d` is correct in the constructor.